webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js

```diff
@@ -120,6 +120,17 @@ export const ErrataTab = () => {
     initialSortColumnName: 'Errata',
   });
 
+  const filtersQuery = () => {
+    let query = [];
+    if (errataTypeSelected !== ERRATA_TYPE) {
+      query.push(`type=${TYPES_TO_PARAM[errataTypeSelected]}`);
+    }
+    if (errataSeveritySelected !== ERRATA_SEVERITY) {
+      query.push(`severity=${SEVERITIES_TO_PARAM[errataSeveritySelected]}`);
+    }
+    return query.join('&');
```

Comment:
I'm not sure if `&` works the same as `and`. And we use `and` in most other places, I think, including `fetchBulkParams`.

```suggestion
    return query.join(' and ');
```